packages/components/src/Emphasis/Emphasis.test.js

```diff
@@ -57,4 +57,13 @@ describe('Emphasis', () => {
 		// then
 		expect(wrapper.find('em').length).toBe(2);
 	});
+
+	it('should emphasize if value is not string', () => {
+		const text = '85';
```

Comment:
```suggestion
		const text = 85;
```

Reply:
then need to convert to string also text, not only value, did it here https://github.com/Talend/ui/pull/3257/files#diff-fc29267dbfa496e8b85223f1144fac0ee40b241143ea905d7e7d36ad27309fcdR20